tools/webpack.config.ts

```diff
@@ -95,7 +95,14 @@ export default (env?: string): webpack.Configuration[] => {
                         {
                             loader: 'css-loader',
                             options: {
-                                importLoaders: 1,
+                                importLoaders: 2,
+                                sourceMap: true,
+                            },
+                        },
+                        {
+                            loader: 'sass-loader',
+                            options: {
+                                implementation: require('sass'),
```

Comment:
@andersk can you review the webpack stuff here?

Reply:
Er, what specifically do you think we want Sass for? We’re using PostCSS rather than Sass; there should be no reason to use both. In general, all Sass features should have PostCSS equivalents; some may need to be explicitly enabled in postcss.config.js.